packages/starter/example/src/components/ContextProvider.tsx

```diff
@@ -71,7 +72,21 @@ const WalletContextProvider: FC<{ children: ReactNode }> = ({ children }) => {
              * instantiate its legacy wallet adapter here. Common legacy adapters can be found
              * in the npm package `@solana/wallet-adapter-wallets`.
              */
-            new UnsafeBurnerWalletAdapter(),
+            new WalletConnectWalletAdapter({
+                network,
+                options: {
+                    relayUrl: 'wss://relay.walletconnect.com',
+                    // example WC app project ID
+                    projectId: 'f1559cce010fd0ba641c3e6e02ca3aa4',
+                    metadata: {
+                        name: 'Example App',
+                        description: 'Example App',
+                        url: 'https://github.com/solana-labs/wallet-adapter',
+                        icons: ['https://avatars.githubusercontent.com/u/35608259?s=200'],
+                    },
+                },
+            }),
+            // new UnsafeBurnerWalletAdapter(),
```

Comment:
Ah, I see these changes are probably just for testing? This package should be reverted before merge.